setup.py

```diff
@@ -22,14 +22,17 @@ def package_data(pkg, roots):
 
 setup(
     name='xblock-leaderboard',
-    version='0.1',
+    version='0.2',
     description='An XBlock which lists the top threads of an inline discussion.',
     packages=[
         'forum_leaderboard',
     ],
     install_requires=[
         'XBlock',
+        'xblock-utils',
+        'ddt',
     ],
+    dependency_links=['https://github.com/edx-solutions/xblock-utils/archive/master.tar.gz'],
```

Comment:
This isn't working for me...

```
Could not find any downloads that satisfy the requirement xblock-utils (from xblock-leaderboard==0.2)
Cleaning up...
No distributions at all found for xblock-utils (from xblock-leaderboard==0.2)
```

Apparently `dependency_links` [is disabled by default in pip, requires a different format](http://stackoverflow.com/questions/12518499/pip-ignores-dependency-links-in-setup-py), and "has been deprecated with an accelerated time schedule and will be removed in pip 1.6".

I just had to manually run `pip install git+https://github.com/edx-solutions/xblock-utils.git` to get it to work.